beacon_node/network/src/sync/block_lookups/mod.rs

```diff
@@ -126,7 +126,16 @@ impl<T: BeaconChainTypes> BlockLookups<T> {
         cx: &mut SyncNetworkContext<T>,
     ) {
         let lookup = self.new_current_lookup(block_root, None, &[peer_source], cx);
+
         if let Some(lookup) = lookup {
+            let blob_indices = lookup.blob_request_state.requested_ids.indices();
+            debug!(
+                self.log,
+                "Searching for block";
+                "peer_id" => ?peer_source,
+                "block" => ?block_root,
+                "blob_indices" => ?blob_indices
```

Comment:
Might be nice to log blob_indices only if they are non-empty for networks that haven't transitioned to deneb

Reply:
good point!

Reply:
Doing this made me realize we may have been artificially delaying single block lookups pre-deneb so I added a check for whether we're in the deneb epoch to `should_delay_lookup`